docs/api/vast-tracker.md

```diff
@@ -136,10 +136,35 @@ Macros will be replaced and the skip tracking URL will be called with the follow
 
 ## Public Methods 💚 <a name="methods"></a>
 
-### errorWithCode(errorCode, isCustomCode)
+### error(macros, isCustomCode)
 
-Sends a request to the URI provided by the VAST `<Error>` element. If an `[ERRORCODE]` macro is included, it will be substituted with `errorCode`.
-Pass `isCustomCode` as true in order to use any value. If false or no value is passed, the macro will be replaced using `errorCode` only if the code is a number between 100 and 999 (see <https://gist.github.com/rhumlover/5747417>). Otherwise 900 will be used.
+Send a request to the URI provided by the VAST <Error> element. `macros` is an optional Object containing macros and their values to be used and replaced in the tracking calls.
+Pass `isCustomCode` as true in order to use any value, `isCustomCode` and its value are related to the[ERRORCODE] macro. If no [ERRORCODE] macro is provided, `isCustomCode`, has no purpose. If false or no value is passed, the macro will be replaced using `errorCode` only if the code is a number between 100 and 999 (see <https://gist.github.com/rhumlover/5747417>). Otherwise 900 will be used.
+
+#### Parameters
+
+- **`macros : Object `** - An optional Object containing macros and their values to be used and replaced in the tracking calls.
+- **`isCustomCode: Boolean`** - Flag to allow custom values on error code.
+
+#### Example
+
+```Javascript
+const customCode = '405';
+
+const macrosParam = {
+  CONTENTURI: 'https://mycontentserver.com/video.mp4',
+  ERRORCODE : customCode 
+}
+
+// Bind error listener to the player
+player.on('error', () => {
+  vastTracker.error(macrosParam);
+});
+```
+
+### errorWithCode(errorCode, isCustomCode) : ⚠️ This method is deprecated in favor of the [error method](https://github.com/dailymotion/vast-client-js/blob/2137a42572da69b1b10b4bbbc30a05dbcceb9083/docs/api/vast-tracker.md#errormacros-iscustomcode)
```

Comment:
Can we add a link (ex: <a name="error"></a>) to the error(macros, isCustomCode) method and use it here as [error method](#error) to be more future proof ?